apps/spruce/src/pages/image/index.tsx

```diff
@@ -1,3 +1,43 @@
-const Image: React.FC = () => <div>{}</div>;
+import { Link, useParams, Navigate } from "react-router-dom";
+import { SideNav, SideNavGroup, SideNavItem } from "components/styles";
+import { ImageTabRoutes, getImageRoute, slugs } from "constants/routes";
+import { getTabTitle } from "./getTabTitle";
+
+const Image: React.FC = () => {
+  const { [slugs.imageId]: imageId, [slugs.tab]: currentTab } = useParams<{
+    [slugs.imageId]: string;
+    [slugs.tab]: ImageTabRoutes;
+  }>();
+
+  // @ts-expect-error: FIXME. This comment was added by an automated script.
+  if (!Object.values(ImageTabRoutes).includes(currentTab)) {
+    return (
+      <Navigate
+        replace
+        // @ts-expect-error: FIXME. This comment was added by an automated script.
```

Comment:
let's change this to do TODO
```suggestion
        // @ts-expect-error: TODO fix in DEVPROD-7654
```